lisa/features/non_ssh_executor.py

```diff
@@ -0,0 +1,53 @@
+from lisa.feature import Feature
+from lisa.features.serial_console import SerialConsole
+
+
+class NonSshExecutor(Feature):
+    """
+    NonSshExecutor is used to run commands on the node when SSH is not available.
+    Lisa by default uses SSH for connection, but this feature provides an alternative
+    execution method for scenarios where SSH connectivity is not possible or desired.
+    """
+
+    COMMANDS_TO_EXECUTE = [
```

Comment:
The commands are not related to `NonSshExecutor`, they should be moved to node level.